Formula/geocouch.rb

```diff
@@ -137,4 +139,6 @@ def caveats; <<-EOS.undent
     run 'brew info geocouch'.)
     EOS
   end
+
+  test {}
```

Comment:
You can just wait until it's merged in to `couchdb`. We'll take care of most of the audit problems as part of that merge, and the code will change enough that you'll need to restart from the new formula, if there's any more work to be done.

What *would* be very helpful is if you could help us devise a test for the `geocouch` component of `couchdb`. Is there something simple we can run against `couchdb` to verify that the `geocouch` extension has been installed and is working? Something "Hello world" level. If you have ideas, drop them in comments on #475.